### ISSUE TITLE
tiles:ignore="true"

### ISSUE THREAD

**Issue by matthaltom** [thymeleaf/thymeleaf-extras-tiles2#17]
It would be really cool if you use tiles:ignore in conjunction with tiles:substituteby.

http://tiles.apache.org/framework/tiles-jsp/tlddoc/tiles/insertAttribute.html

**Comment by Macs75** [comment]
I agree, just started now and found this feature missing, really a pity.

**Comment by php-coder** [comment]
Yes, this is why I decided to not use this project =/
